appdev/guides/dependencies.rst

```diff
@@ -53,15 +47,17 @@ Optionally, configure the compilation by adding ``build_args``, which may look l
       }
     }
 
-Build arguments are usually project specific. Therefore, study the library's build instructions and also look for ``option`` settings in its ``CMakeLists.txt`` (if CMake is used). The default build directory is ``build/LIBNAME/ARCHITECTURE``. Therefore builds for different architectures can exist in parallel.
+Most build arguments are project specific. Therefore, study the library's build instructions and also look for ``option`` settings in its ``CMakeLists.txt`` .
 
-To actually build the library run ``clickable build-libs``. You may want to build it for the desktop, too, via ``clickable build-libs --arch amd64``. Don't forget to mention this step in your README!
+To actually build the library for all architectures run ``clickable build-libs --arch armhf``, ``clickable build-libs --arch arm64`` and ``clickable build-libs --arch amd64``. Don't forget to mention this step in your README, so that others an reproduce the build process.
 
-See how `Teleports clickable.json <https://gitlab.com/ubports/apps/teleports/blob/master/clickable.json#L14>`_ uses the libraries feature to build its dependency tdlib.
+See how `Teleports clickable.json <https://gitlab.com/ubports/apps/teleports/blob/master/clickable.json#L21>`_ uses the libraries feature to build its dependency tdlib.
 
 Pre-built
 ^^^^^^^^^
-A pre-built library is usually only available as a shared object, that needs to be linked dynamically. Furthermore, it may contain components that you don't need, resulting in a bloated app. It may even miss something, that you could achieve by compiling it yourself. Sometimes, a library is available in the Ubuntu Repositories, but is not installable for the architecture you need (likely ``armhf``). In this case you have to compile the library as described above.
+If you compiled your library as described above, skip this step.
+
+A pre-built library is usually only available as a shared object, that needs to be linked dynamically. Furthermore, it may contain components that you don't need, resulting in a bloated app. It may even miss something, that you could achieve by compiling it yourself. Sometimes, a library is available in the Ubuntu Repositories, but is not installable for the architecture you need (likely ``armhf`` or ``arm64``). In this case you have to compile the library as described above.
```

Comment:
"Shared object that needs to be linked dynamically" (Remove comma)

"It may even miss a feature that you could add if you were compiling it yourself."

Reply:
Good to have a native speaker here. :)